python/tk_unreal/unreal_sg_engine.py

```diff
@@ -126,12 +127,12 @@ def _get_context(self, engine):
                     try:
                         # Entity id must be converted to an integer
                         entity_id = int(tokens[-1])
-                    except:
+                    except Exception:
```

Comment:
Is ok not to log this?

Reply:
I don't know, I just fixed the bare except which wouldn't pass flake8.